src/uct/sm/cma/cma_pd.h

```diff
@@ -0,0 +1,91 @@
+/**
+* Copyright (C) Mellanox Technologies Ltd. 2001-2015.  ALL RIGHTS RESERVED.
+* Copyright (c) UT-Battelle, LLC. 2014-2015. ALL RIGHTS RESERVED.
+*
+* $COPYRIGHT$
+* $HEADER$
+*/
+
+#ifndef UCT_CMA_PD_H_
+#define UCT_CMA_PD_H_
+
+#include <ucs/config/types.h>
+#include <ucs/debug/memtrack.h>
+#include <ucs/type/status.h>
+#include <uct/tl/context.h>
+
+#include <sys/types.h>
+#include <unistd.h>
+
+/*
+ * Define a memory-mapper component for MM.
```

Comment:
yeah,  I missed it,  but this part should not be copied from mm.  cma_pd should be only NOPs.